DetectorDescription/DDCMS/interface/MuonNumberingRcd.h

```diff
@@ -0,0 +1,11 @@
+#ifndef GEOMETRY_RECORDS_MUON_NUMBERING_RCD_H
+#define GEOMETRY_RECORDS_MUON_NUMBERING_RCD_H
+
+#include "FWCore/Framework/interface/DependentRecordImplementation.h"
+#include "DetectorDescription/DDCMS/interface/DetectorDescriptionRcd.h"
+#include "DetectorDescription/DDCMS/interface/DDSpecParRegistryRcd.h"
+#include "boost/mpl/vector.hpp"
+
+class MuonNumberingRcd : public edm::eventsetup::DependentRecordImplementation<
+MuonNumberingRcd, boost::mpl::vector<DDSpecParRegistryRcd, DetectorDescriptionRcd>> {};
```

Comment:
Why declare `MuonNumberingRcd` to be dependent on `DetectorDescriptionRcd` if the ESProducers does not care about the latter record type?

Reply:
It’s what we have now with IdealGeometry and MuonNumberingRecord. I’ll add a note to revise current records.